contrib/index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Have fun ♥️</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+
+    <header>
+      <h1>Hey there! Thanks for contributing to <a href="/">Bitters</a></h1>
+    </header>
+
+    <main>
+
+      <div class="wrapper">
```

Comment:
Actually, `wrapper` isn’t use in the styles…we should add it to `styles.scss` and then wrap everything within `body`.